docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - ./keys:/root/.ssh
       - ssh:/retool_backend/autogen_ssh_keys
       - ./retool:/usr/local/retool-git-repo
+      - ${BOOTSTRAP_SOURCE}:/usr/local/retool-repo
```

Comment:
also covering this https://github.com/tryretool/retool-onpremise/pull/96 (so whatever we merge first)